src/renderer/components/button/button.tsx

```diff
@@ -26,36 +25,33 @@ export interface ButtonProps extends ButtonHTMLAttributes<any>, TooltipDecorator
   target?: "_blank"; // in case of using @href
 }
 
-@withTooltip
-export class Button extends React.PureComponent<ButtonProps, {}> {
-  render() {
-    const {
-      waiting, label, primary, accent, plain, hidden, active, big,
-      round, outlined, tooltip, light, children, tooltipOverrideDisabled, ...btnProps
-    } = this.props;
+export const Button = withTooltip((props: ButtonProps) => {
+  const {
+    waiting, label, primary, accent, plain, hidden, active, big,
+    round, outlined, light, children, ...btnProps
+  } = props;
 
-    if (hidden) return null;
+  if (hidden) return null;
 
-    btnProps.className = cssNames("Button", btnProps.className, {
-      waiting, primary, accent, plain, active, big, round, outlined, light,
-    });
+  btnProps.className = cssNames("Button", btnProps.className, {
+    waiting, primary, accent, plain, active, big, round, outlined, light,
+  });
 
-    // render as link
-    if (this.props.href) {
-      return (
-        <a {...btnProps}>
-          {label}
-          {children}
-        </a>
-      );
-    }
-
-    // render as button
+  // render as link
```

Comment:
Yeah, but why? :) Rendering something but a button when the component is called `Button` is wild. It also breaks the HTML-semantics.